acra/configuring-maintaining/installing/building-acrawriter.md

```diff
@@ -122,16 +140,13 @@ work
 
 #### Installation
 
-```bash
-sudo apt install php5 php5-dev
-cd themis
-sudo make phpthemis_install
-# extension=phpthemis.so
-sudo nano /etc/php5/cli/php.ini
-# check that extension is available
-php --ri phpthemis
-cd -
-```
+Install [phpthemis extension]({{< ref "/themis/languages/php/installation.md#building-latest-version-from-source" >}})
+
+{{< hint info >}}
+AcraWriter for PHP is currently not packaged because requires a binary extension. It is available only in source form and supports only PHP 5 & 7 versions
+{{< /hint >}}
```

Comment:
```suggestion
{{< hint info >}}
AcraWriter is currently not packaged for PHP.
It is available in source form and requires PHPThemis extension to be installed.
PHP versions 5.6, 7.0–7.2 are supported.
{{< /hint >}}

Please follow [PHPThemis installation instructions]({{< ref "/themis/languages/php/installation.md" }}).
```

AcraWriter *can* be packaged just like PHPThemis is packaged. The only reason it's not packaged is the maintainer capacity.